--- cluster-2.00.00/fence/fenced/Makefile~	2007-04-03 15:10:16.000000000 +0000
+++ cluster-2.00.00/fence/fenced/Makefile	2007-04-03 15:13:45.221772000 +0000
@@ -35,9 +35,8 @@
 	group.o \
 	member_cman.o \
 	agent.o \
-	../../ccs/lib/libccs.a \
 	../../group/lib/libgroup.a
-	$(CC) $(LDFLAGS) -o $@ $^ -lcman 
+	$(CC) $(LDFLAGS) -o $@ $^ -lcman -lccs
 
 main.o: main.c fd.h
 	$(CC) $(CFLAGS) -c -o $@ $<
